docs/the-contribution-workflow.md

```diff
@@ -0,0 +1,245 @@
+## Contributing to Exercism
+
+Open source contributions can be intimidating! As you're looking around, wondering where to dive in, you might think everyone _else_ knows what's going on in the code base, or has better grasp of the language, or seems to be well known around the project.
+
+As a total beginner, where do you begin?
+
+Right here. This guide will walk you through your first open source contribution.
```

Comment:
And this is a great continuation of that opening. It's super welcoming and friendly.